src/Discord.Net.Commands/Attributes/DescriptionAttribute.cs

```diff
@@ -12,7 +13,8 @@ public DescriptionAttribute(string text)
         }
     }
 
-    [AttributeUsage(AttributeTargets.Method)]
+    // Brief summary of method/module
```

Comment:
Uhhhh.... Synopsis was intended to be the Longer summary, as Description is for Parameters, too.... A Synopsis is something intended to give an overview of something, beginning to end.